AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -365,6 +620,12 @@ private MouseMode CurrentMode
             }
             set
             {
+                if (_currentMode == MouseMode.DragAll && value != MouseMode.DragAll)
+                {
+                    // Finishing a drag all operation, add a NoOp action so later ones do not combine.
+                    actionManager.PerformAction(new NoOpAction());
```

Comment:
What happens if I move one object, and then I move another one? Will these two undoable operations be merged into single one?

Reply:
Ah so I haven't yet implemented moving selected objects only all of them at once (what happens when you `LEFT` + `RIGHT` click at the same time.